packages/web/src/scss/helpers/links/index.html

```diff
@@ -84,6 +84,27 @@ <h2 class="docs-Heading">Underlined & Disabled</h2>
 
 </section>
 
+<section class="UNSTABLE_Section">
+
+  <h2 class="docs-Heading">Not Underlined</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+    <p>
+      <a href="#" class="link-primary link-not-underlined">Primary Disabled Underlined Link</a>
+    </p>
+    <p>
+      <a href="#" class="link-secondary link-not-underlined">Secondary Disabled Underlined Link</a>
+    </p>
+    <p class="docs-Box">
+      <a href="#" class="link-inverted link-not-underlined">Inverted Disabled Underlined Link</a>
+    </p>
+    <p>
+      <button type="button" class="link-primary link-not-underlined" disabled>Link as button</button>
```

Comment:
this doesn't work. it is still underlined on hover.

Reply:
Hm, fixed...